app.rb

```diff
@@ -38,6 +40,11 @@ def quality_indicator_svg
     def quality_indicator_group input
       input >= 100 ? "5" : (input / 20) + 1
     end
+
+    def localize_number(n)
+      # Explicitly do US locale for now since nothing else is localized
+      ActiveSupport::NumberHelper.number_to_delimited(n, locale: 'en-US')
```

Comment:
Is it? Well `en-GB` and `en-US` are the same as far as number formatting is concerned so